test/e2e/apps/types.go

```diff
@@ -40,11 +40,11 @@ var (
 	// KittenImage is the fully qualified URI to the Kitten image
 	KittenImage = imageutils.GetE2EImage(imageutils.Kitten)
 
-	// NginxImage is the fully qualified URI to the Nginx image
-	NginxImage = imageutils.GetE2EImage(imageutils.Nginx)
+	// WebserverImage is the fully qualified URI to the Httpd image
+	WebserverImage = imageutils.GetE2EImage(imageutils.Httpd)
 
-	// NewNginxImage is the fully qualified URI to the NginxNew image
-	NewNginxImage = imageutils.GetE2EImage(imageutils.NginxNew)
+	// NewWebserverImage is the fully qualified URI to the Httpd22 image
+	NewWebserverImage = imageutils.GetE2EImage(imageutils.Httpd22)
```

Comment:
the name assigned to Httpd22 seems to be NewWebserverImage
but `2.2-alpine` is probably older than `2.4-alpine`?

Reply:
yeah, I have to read the test code again, don;t know if is new as in different or new as in newer